packages/terra-date-picker/src/DatePicker.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+@import '../node_modules/react-datepicker/dist/react-datepicker.min';
```

Comment:
We can't always guarantee that this relative path will work. I would update the syntax to 

@import '~react-datepicker/dist/react-datepicker.min';

This syntax is from the sass-loader we use with webpack: https://github.com/webpack-contrib/sass-loader#imports